model/riscv_types_kext.sail

```diff
@@ -68,8 +68,14 @@ function aes_mixcolumn_inv(x) = {
   b3 @ b2 @ b1 @ b0 /* Return value */
 }
 
-val aes_decode_rcon : bits(4) -> bits(32)
+/* Turn a round number into a round constant for AES. Note that the
+   AES64KS1I instruction is defined such that the r argument is always
+   in the range 0x0..0xA. Values of rnum outside this range do not decode
+   to the AES64KS1I instruction. The 0xA => 0x0 case is used specifically
```

Comment:
This special case should no longer be hit thanks to the revised AES64KS1I implementation, so you can have an assert that it's less than 0xA

Reply:
Ah right you are! Duly change in the most recent push. :+1: